dragon/runtime_docs.rb

```diff
@@ -278,11 +278,11 @@ def tick args
 - ~:down_or_held~
 - ~:up~
 ** ~args.outputs~
-~args.outputs.PROPERTY~ is how you render to the screen.
+~args.outputs.PROPERTY~ is how you render to the screen and play sounds. Rendering to the screen happens in the following order: ~solids < sprites < primitives < labels < lines < borders < debug~ and ~static_*~ are drawn above their non-static counterparts.
```

Comment:
Can we have "solids < sprites < primitives < labels < lines < borders < debug~ and ~static_*" in a new line - so that folks don't miss it. On the other hand, i am assuming you wanted to make this bold, but right now it appears as strike through like below.

~solids < sprites < primitives < labels < lines < borders < debug~ and ~static_*~

Reply:
This brings up a good question, actually. In markdown, `~strike~` turns into ~strike~, however here you'll see that tildes are used around code (see earlier on the same line `~args.outputs.PROPERTY~` which appears monospaced in the final output, and is where I copied the tilde style from). So what markup language are these docs actually using? The only one my quick search found was [Org-Mode](https://en.wikipedia.org/wiki/Org-mode) which seems like a strange choice.

Reply:
I am now convinced that this is indeed using Org-Mode for markup.